fractal_server/config.py

```diff
@@ -419,4 +419,7 @@ def check(self):
 
 
 def get_settings(settings=Settings()) -> Settings:
+    logging.warn("Fractal Settings:")
+    for k, v in settings.dict().items():
+        logging.warn(f"{k}: {v}")
```

Comment:
Maybe we should replace `v` with `"***"` when `k` includes strings like `"PASSWORD"` or `"SECRET"`?

This is because, until we tackle #620, it is not 100% clear where logs are ending up - since it depends on which startup command is used.